src/wp-includes/cache.php

```diff
@@ -302,15 +302,15 @@ function wp_cache_flush_group( $group ) {
 }
 
 /**
- * Determines whether the object cache implementation supports flushing individual cache groups.
+ * Determines whether the object cache implementation supports a particular feature.
  *
  * @since 6.1.0
  *
- * @see WP_Object_Cache::flush_group()
- *
- * @return bool True if group flushing is supported, false otherwise.
+ * @param string $feature Name of the feature to check for.
+ *                        Possible values include: 'flush_group'.
+ * @return bool True if the feature is supported, false otherwise.
  */
-function wp_cache_supports_group_flush() {
+function wp_cache_supports( $feature ) {
 	return true;
```

Comment:
Maybe we add the framework already?

```suggestion
	switch ( $feature ) {
		'flush_group':
		    return true;
		default:
		    return false;
	}
```

Reply:
I wonder if we should add, get_multiple, delete_mulitple, update_multiple and flush_runtime

Reply:
Yeah, exactly 👍

Reply:
Added the `switch` statement. :+1:

@tillkruss @spacedmonkey @JJJ If we add the other features here, do we need to make any changes to the functions in `wp-includes/cache-compat.php` to check for `wp_cache_supports()`? 

It looks like these functions are polyfilled anyway, so we could add these features in `wp_cache_supports()`, but the functions themselves would not need any changes:
* `wp_cache_add_multiple()`
* `wp_cache_set_multiple()`
* `wp_cache_get_multiple()`
* `wp_cache_delete_multiple()`

`wp_cache_flush_runtime()`, however, may need an `if ( ! wp_cache_supports( 'flush_runtime' ) )` check similar to the one in `wp_cache_flush_group()`:
```
if ( ! wp_cache_supports( 'flush_runtime' ) ) {
	_doing_it_wrong(
		__FUNCTION__,
		__( 'Your object cache implementation does not support flushing the in-memory runtime cache.' ),
		'6.1.0'
	);

	return false;
}
```

Reply:
Yeah, I was mostly thinking of 3rd party plugins correctly listing their supported features.